src/software/physics/euclidean_to_wheel_test.cpp

```diff
@@ -184,16 +183,15 @@ TEST_F(EuclideanToWheelTest, test_velocity_no_ramping)
     auto current_wheel_velocity_coefficient = std::min(
         max_allowable_wheel_velocity * 0.9, allowable_delta_wheel_velocity * 0.9);
```

Comment:
Im a bit confused by what this is calculating. We're comparing delta velocity (acceleration in 1 iteration) and max overall velocity? Seems like mismatching units. Similarly for the next couple of tests

Reply:
so basically i did this to make the test work despite the values of max acceleration or max overall velocity changing. 

So the current velocity in that test is 0, so whatever the target velocity is, that's the delta. So to avoid the first ramping, the target velocity's max coeff has to be lower than the max delta

To avoid the second check, the target velocity's max coeff has to be lower than the max overall velocity

So I just took the min of them to get the target velocity coefficient. So it is 2 different units (acceleration vs velocity), but since the current velocity is 0, the target velocity is both the delta and the overall velocity